### PR TITLE
Add example folder with example config file and output

### LINKED ISSUE #36: There is no example scenario with sample files and example config files

### PR DESCRIPTION
**Pull request by dog-ball**
Fixes #36

### CODE REVIEW

File: examples/example_scenario/matches.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
+time,bid_id,ask_id,bid_actor,ask_actor,bid_cluster,ask_cluster,energy,price,grid_fee
+8.0,1,2,H1,H2,1,0,0.1,0.0945287370914458,0.010000000000000002
+8.0,1,0,H1,H0,1,0,0.2,0.0945287370914458,0.010000000000000009
+9.0,1,0,H1,H0,1,0,0.30000000000000004,0.018104619792651627,0.010000000000000002
+10.0,1,0,H1,H0,1,0,0.2,0.08884487582548159,0.010000000000000002
+10.0,1,2,H1,H2,1,0,0.1,0.08884487582548159,0.010000000000000002
+10.0,4,2,H4,H2,1,0,0.2,0.08884487582548159,0.010000000000000002
+10.0,4,3,H4,H3,1,0,0.2,0.08884487582548159,0.010000000000000009
```
Comment (j-ti) [inline examples/example_scenario/matches.csv:8]: Will be overridden -> rename for validation

File: examples/example_scenario/orders.csv [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+-1.0,8,H4,1,0.30000000000000004,0.053156299468968306
+1.0,9,H0,0,0.5,0.008104619792651625
+-1.0,9,H1,1,0.30000000000000004,0.03785467974641451
+1.0,9,H2,0,0.1,0.0557650755696901
+1.0,9,H4,1,0.1,0.0363878309877054
+1.0,10,H0,0,0.2,0.000858549110458079
+-1.0,10,H1,1,0.30000000000000004,0.22795652435352368
+1.0,10,H2,0,0.30000000000000004,0.012779532496533963
+1.0,10,H3,0,0.30000000000000004,0.07884487582548158
+-1.0,10,H4,1,0.4,0.16249843482194495
```
Comment (j-ti) [inline examples/example_scenario/orders.csv:15]: Will be overridden -> rename for validation

**Review by j-ti — CHANGES_REQUESTED** [review]
see comments

**Comment by dog-ball** [comment]
I have renamed the example scenario to validation scenario so it is no longer overwritten when the example config file is run. Maybe I should reference the examples folder in the documentation so people understand its use?

**Review by j-ti — CHANGES_REQUESTED** [review]
with the example config only matches.csv and orders.csv get overridden by an execution, so a suffix to the file is sufficient rather than a new folder, the other files should stay in example_scenario while the config already guarantees that they are not updated, i.e. `update_scenario = false`

File: examples/config_example.txt [hunk 3]
```diff
@@ -0,0 +1,14 @@
+[default]
+show_plots = true
+show_prints = true
+path = examples/example_scenario
+nb_ts = 3
+save_csv = true
+# If orders should be saved, the market should not be reset
```
[the remark this answers is not in the document]
Comment (j-ti) [reply examples/config_example.txt:7]: comment still exists

**Pull request merged**